fmutils_test.go

```diff
@@ -768,6 +768,128 @@ func TestPrune(t *testing.T) {
 	}
 }
 
+func TestOverride(t *testing.T) {
+	tests := []struct {
+		name  string
+		paths []string
+		src   proto.Message
+		dest  proto.Message
+		want  proto.Message
+	}{
+		{
+			name: "profile override",
```

Comment:
Please add a better explanation for this test case. It's unclear what result is expected

Reply:
This test was combination of bunch of things, will separate it to few tests with better names